llvm/lib/Transforms/Utils/InjectTLIMappings.cpp

```diff
@@ -35,35 +35,36 @@ STATISTIC(NumCompUsedAdded,
 
 /// A helper function that adds the vector function declaration that
 /// vectorizes the CallInst CI with a vectorization factor of VF
```

Comment:
nit: could improve original doxygen comment with `\p CI` instead of `CI`.
Same for `VF`.

Reply:
Done